app/views/items/index.html.haml

```diff
@@ -55,6 +56,5 @@
                 .meta $ #{item.price} #{" | #{item.special_date} Special" if item.special_date}
                 .description= item.description
               .extra.content
-                %span.left.floated= item.category.capitalize if item.category
                 %span.right.floated 
-                  =link_to "Add", item_selections_path(item), method: :post, id: "#{dom_id(item)}_add"
\ No newline at end of file
+                  =link_to "Add", item_selections_path(item), method: :post, id: "#{dom_id(item)}_add", class: "ui label #{'disabled' unless item.currently_available?}"
```

Comment:
Not a big issue, but the disabled button still has a darkening transition, which doesn't make it clear it's not clickable. Maybe making it a `button` class instead of a `label`.